Cython/Utils.py

```diff
@@ -543,6 +543,7 @@ def print_captured(captured, output, header_line=None):
     if captured:
         if header_line:
             output.write(header_line)
+        captured = captured.encode(errors='replace').decode()
```

Comment:
> `UnicodeEncodeError: 'ascii' codec can't encode characters in position 105-107: ordinal not in range(128)`

I think the code line that you added is the wrong way to deal with this issue. In Python 3, it probably doesn't do anything because the default encoding tends to be UTF-8, for which encoding+decoding is a no-op. In Python 2, this is only needed if the output (or default?) encoding cannot handle (non-ascii) Unicode text. I don't actually know why it encodes (to 'ascii') here. The logs are gone, so I cannot look up the IPython/Jupyter version that was in use. You were using an older 5.x version locally, so maybe that's a difference? I cannot reproduce it with an 8.x version either though (on Linux).

Reply:
If I remember correctly, this was a CI issue and nobody could reproduce it locally

Reply:
OK. Now I tried to remove this line. Let's see what will say CI...

Reply:
Hmm I tried to remove the line and now the CI works fine. Not sure what was the issue.